app/server.py

```diff
@@ -76,6 +76,41 @@ def index():
 
 @app.route('/inventory', methods=[GET])
 def query_prod_info():
+    """
+    Retrieve a list of inventory items
+    This endpoint will return all details of the all the products in the inventory unless a query parameter is specificed
+    ---
+    tags:
+      - inventory
```

Comment:
Please unify the indentation style, refer to swagger doc starting from line 139.

Reply:
sure